src/Sylius/Behat/Client/Helper.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Behat\Client;
+
+use Sylius\Behat\Service\SharedStorageInterface;
+use Webmozart\Assert\Assert;
+
+final class Helper
+{
+    public function __construct(
+        private SharedStorageInterface $sharedStorage,
+        private RequestBuilderFactoryInterface $requestBuilderFactory,
+        private ApiClientInterface $apiClient,
+        private string $authorizationHeader,
+    ) {
+    }
+
+    public function update(callable $closure): void
+    {
+        /** @var RequestBuilder $requestBuilder */
+        $requestBuilder = $this->sharedStorage->get('request_builder');
+
+        $requestBuilder = $closure($requestBuilder);
+
+        $this->sharedStorage->set('request_builder', $requestBuilder);
+    }
+
+    public function authorize(RequestBuilder $requestBuilder): RequestBuilder
```

Comment:
This method should be done automatically if we are authorized(we called the "I am logged in as..." step in behat). Perhaps it should be done as a wrapper on request factory

Reply:
The SharedStorage sets Bearer token. The token is null when the account is not logged in so the header will be empty